webapp/src/Service/AwardService.php

```diff
@@ -2,12 +2,16 @@
 
 namespace App\Service;
 
+use App\DataTransferObject\Award;
 use App\Entity\Contest;
 use App\Entity\Team;
 use App\Utils\Scoreboard\Scoreboard;
 
 class AwardService
 {
+    /**
+     * @var array<int, Award[]>
```

Comment:
```suggestion
     * @var array<Award[]>
```

Reply:
I did it this way since the array indices are not consecutive but the contest ID's